### ISSUE TITLE
.NET MAUI app's UI Automation hierarchy has a root window element with no accessible name  

### ISSUE THREAD

**Issue by gbarkerz** [dotnet/maui#12973]
### Description

Accessibility Insights for Windows reports a failure with any new .NET 7 Maui app because at the root of the UI Automation hierarchy, there's a window element with no accessible name. I'm assuming this nameless element is the reason why Visual Studio's new integrated Accessibility Checker can't say a MAUI app's UI is problem-free. 

### Steps to Reproduce

1. Create a new MAUI app in VS. 
2. Run the app and run Accessibility Insights for Windows's tests at the app.

Result: Accessibility Insights for Windows says the app window needs an accessible name, as shown in the following image.

![image](https://user-images.githubusercontent.com/77085891/215257906-8fe5f3c0-5841-4eb3-8832-8498800ded49.png)

3. Run VS's integrated Accessibility Checker against the MAUI app.

Result: VS's Accessibility Checker reports that the app has a nameless element. As shown in the following image. I assume that element is the nameless window.

![image](https://user-images.githubusercontent.com/77085891/215257898-609733dd-2755-4ee1-98ab-29ff71f0fdd2.png)

### Link to public reproduction project repository

https://github.com/gbarkerz/MauiAppA

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

net7.0-windows10.0.19041.0

### Did you find any workaround?

No.

**Comment by PureWeen** [comment]
@gbarkerz if you set a title on the `AppShell` does that fix? https://github.com/gbarkerz/MauiAppA/blob/master/MauiAppA/AppShell.xaml#L2

This might possibly be related to https://github.com/microsoft/microsoft-ui-xaml/issues/7853

**Comment by gbarkerz** [comment]
Hi @PureWeen, adding a AppShell Title does indeed fix this! Great news, thanks very much! I've just added the change to the demo MauiAppA, and neither VS nor Accessibility Insights for Windows reports a failure with the app now.

**Comment by jsuarezruiz** [comment]
@PureWeen Review if the Shell Title is used in the project template.

**Comment by hartez** [comment]
> @PureWeen Review if the Shell Title is used in the project template.

And if it _used to_ be there, and it's been removed, we need to investigate why.
